trac-env/templates/site_head.html

```diff
@@ -1,3 +1,4 @@
+<meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
Seems like this could also be extracted to its own commit/PR and be merged independently, right?

Reply:
Extracted: https://github.com/django/code.djangoproject.com/pull/212